.github/workflows/clean.yml

```diff
@@ -0,0 +1,50 @@
+name: Clean
```

Comment:
If you pardon my curiosity, what is the purpose of this?

Reply:
This is actually auto-generated stuff from Daniel's plugin, intended to clean artifacts produced by build.

Reply:
Yeah, but why would you want to do that? The whole CI environment disappears after it finishes, so why bother deleting individual files?

Reply:
We can clean it up.